markov_chain.py

```diff
@@ -0,0 +1,121 @@
+"""Generates a pseudo-random string of text using a markov chain language model and a text file source ."""
```

Comment:
If you sourced this file from another project and are just including it directly, please include a comment with a URL of its source.